docker/developers/docker-compose.mysql.yml

```diff
@@ -0,0 +1,41 @@
+include:
+  - path: ./docker-compose.networks.yml
+
+services:
+  # Docker-managed database
+  mysql-base: &mysql-base
+    hostname: db
+    image: mysql/mysql-server:8.0
+    # You can add default options via the 'command':
+    #command:
+    # - '--log-error-verbosity=3'
+    environment:
+      MYSQL_DATABASE: 'db'
+      MYSQL_USER: 'user'
+      MYSQL_PASSWORD: 'password'
+      MYSQL_ROOT_PASSWORD: 'password'
```

Comment:
Instead of having a defined password, would it make sense to enable "allow empty password"?

Reply:
Why? It doesn't make anything simpler, does it?